pages/0_cadeaubon.php

```diff
@@ -289,11 +289,11 @@ function createRandomCode() {
             </tr>
             <tr>
                 <td>Op naam van (verplicht):</td>   
-                <td><input type="text" name="InNameOf" value="<?=$userName?>"></td>
+                <td><input type="text" name="InNameOf" value="<?=$userName?>"required/></td>
             </tr>
             <tr>
                 <td>E-mail adres (verplicht):</td>
-                <td><input type="text" name="Email" value=<?=$userEmail?>></td>
+                <td><input type="text" name="Email" value=<?=$userEmail?> required/></td>
```

Comment:
zie regel 292, moet dit ook niet tussen " " of moet het op regel 292 juist zonder " " er omheen?